src/Components/Notifications/PartnerOfferArtwork.tsx

```diff
@@ -130,15 +148,35 @@ export const PartnerOfferArtwork: FC<PartnerOfferArtworkProps> = ({
           </Text>
         </Box>
       )}
-      <Box mb={4} width="100%" maxWidth={CARD_MAX_WIDTH}>
+      <Box
+        mb={4}
+        display="flex"
+        justifyContent="space-between"
+        width="100%"
+        maxWidth={CARD_MAX_WIDTH}
+      >
         <Button
           // @ts-ignore
           as={RouterLink}
           to={href}
           data-testid="partner-offer-artwork-button"
+          mr={partnerOfferVisibilityEnabled && fullyAvailable ? 2 : 0}
```

Comment:
I think you can use the `gap` property on the box above for margins

Reply:
Cool, I'll add it to the follow up changes, thanks!